bundle/config/workspace.go

```diff
@@ -91,6 +88,13 @@ func (w *Workspace) Client() (*databricks.WorkspaceClient, error) {
 		}
 	}
 
+	if w.Profile != "" && cfg.Host != "" {
```

Comment:
Above uses `w` only, this both `w` and `cfg`. Let's use one or the other, not both.